utils/meta_infos.json

```diff
@@ -523,6 +580,50 @@
       "ModelArguments": {
         "hierarchical": true
       }
+    },
+    "swiss_criticality_prediction_bge_facts": {
+      "DataTrainingArguments": {
+        "max_seq_length": 512,
+        "max_segments": 16,
+        "max_seg_length": 128,
+        "add_oversampling": true
+      },
+      "ModelArguments": {
+        "hierarchical": true
+      }
+    },
+    "swiss_criticality_prediction_bge_considerations": {
+      "DataTrainingArguments": {
+        "max_seq_length": 512,
+        "max_segments": 16,
+        "max_seg_length": 128,
+        "add_oversampling": true
+      },
+      "ModelArguments": {
+        "hierarchical": true
+      }
+    },
+    "swiss_criticality_prediction_citation_facts": {
+      "DataTrainingArguments": {
+        "max_seq_length": 512,
+        "max_segments": 16,
+        "max_seg_length": 128,
+        "add_oversampling": true
+      },
+      "ModelArguments": {
+        "hierarchical": true
+      }
+    },
+    "swiss_criticality_prediction_citation_considerations": {
+      "DataTrainingArguments": {
+        "max_seq_length": 512,
+        "max_segments": 16,
+        "max_seg_length": 128,
+        "add_oversampling": true
+      },
+      "ModelArguments": {
+        "hierarchical": true
+      }
```

Comment:
Is there a way to make all of this less "boilerplaty"?

Reply:
@JoelNiklaus before that, as far as I remember, we saved these information in main.py. I thought this was a nice solution to define these arguments for each dataset.